src/locale/it.json

```diff
@@ -650,7 +650,7 @@
       "confirmButton": "Conferma"
     },
     "alert": {
-      "noDelegations": "Tutti gli enti disponibili sono già associati alla stazione selezionata."
+      "noDelegations": "Nessuna delega disponibile per la stazione selezionata. Visualizzi questo messaggio se non hai deleghe oppure se tutti gli enti che ti hanno delegato sono già associati a questa stazione"
```

Comment:
this should not be committed, try to rebase with origin/netx branch

Reply:
should be updated